.gitattributes

```diff
@@ -1,15 +0,0 @@
-# Auto detect text files and perform LF normalization
```

Comment:
It caused git clone to autoconvert some files (no matter which OS or method it seems). I think @yohanmishkin had the same problem. Perhaps it can be adjusted rather than removed. The default .gitattributes that Visual Studio 2015 creates has only `* text=auto` so perhaps that would work and be the "correct" way to handle it.